JSDemos/Demos/Charts/Candlestick/description.md

```diff
@@ -1 +1,15 @@
-In&nbsp;this demo, the **candle stick** series displays variations in&nbsp;stock price throughout the course of&nbsp;a&nbsp;given day. Each point consists of&nbsp;a&nbsp;rectangle (body) whose bottom and topmost values correspond to&nbsp;the opening and closing price of&nbsp;a&nbsp;stock. A&nbsp;vertical line (shadow, wick or&nbsp;tail) displays the high and low price for the stock. If&nbsp;the stock closes higher than its opening price, no&nbsp;color fills its body and if&nbsp;the stock closes lower than its opening price, a&nbsp;color fill is&nbsp;applied.
\ No newline at end of file
+The Candlestick financial Chart is designed to analyze trading patterns over a short period of time. A single point on a Candlestick [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day. 
```

Comment:
Chart shouldn't start with a capital letter in this case.
The chart is not designed to analyze, but rather communicate those trading patterns. Humans do the analysis. 
```suggestion
The Candlestick financial chart is designed to communicate trading patterns over a short period of time. A single point [point? but I see rectangles and lines, but no points] on a Candlestick [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day.
```

Reply:
I meant "data point" here, each point on a line in a [line chart](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/Spline/React/Light/), each bar in a [bar chart](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/StandardBar/React/Light/), each candlestick in a candlestick chart - all that things are data points, like each data object is represented by a data point in a Chart. And when you combine related data points together, they become series. Should I say "data point" instead? We just use simply "point" in our docs when referring to data points.

Reply:
Yes, definitely say data point. :)